lib/matplotlib/backends/backend_pdf.py

```diff
@@ -767,8 +765,6 @@ def __init__(self, filename, metadata=None):
 
     fontNames = _api.deprecated("3.11")(property(lambda self: self._fontNames))
     dviFontInfo = _api.deprecated("3.11")(property(lambda self: self._dviFontInfo))
-    type1Descriptors = _api.deprecated("3.11")(
-        property(lambda self: self._type1Descriptors))
```

Comment:
This isn't private, so we need to keep the property for the deprecation, even if it just returns an empty dictionary.